src/frontend/src/pages/login/login.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Container from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+
+import { Typography } from '@material-ui/core';
```

Comment:
```suggestion
import Typography from '@material-ui/core/Typography';
```
Please use a consistent import style.